doc/api/v0.yaml

```diff
@@ -57,12 +57,12 @@ paths:
         "404":
           description: The requested fund was not found
 
-  /api/v0/proposals:
+  /api/v0/proposals/{voter_group_id}:
     post:
       summary: Get proposal by id
       tags: [ proposal ]
       description: |
-        Retrieves queried proposals.
+        Retrieves queried proposals for the provided voter group identifier.
```

Comment:
what about having two endpoints for:
* proposals by voteplan
* voteplans for voting group
Otherwise we might risk these two to be out of sync

Reply:
We're in a trap of using the same URL for `GET` and `POST`. For now I only specified `voter_group_id` to be only used in `GET`.